service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/sdo/SDONotifyData.java

```diff
@@ -1,20 +1,19 @@
 package uk.gov.hmcts.reform.fpl.model.notify.sdo;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import lombok.Builder;
-import lombok.Data;
+import lombok.Value;
 import uk.gov.hmcts.reform.fpl.model.notify.NotifyData;
 
 import java.util.Map;
 
-@Data
+@Value
 @Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class SDONotifyData implements NotifyData {
-    private final String familyManCaseNumber;
-    private final String leadRespondentsName;
-    private final String hearingDate;
-    private final String reference;
-    private final String caseUrl;
-    private final String title;
-    private final Map<String, Object> documentLink;
-    private final String callout;
+    String leadRespondentsName;
+    String respondentLastName; // only here because the ctsc was designed with this instead of leadRespondentsName
```

Comment:
👍 comment can remind someone to update this if the template ever needs modifying